Workbooks/UpdateCompliance/TargetBuildList/Update compliance _ Target build.workbook

```diff
@@ -0,0 +1,201 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "03761a72-d969-426b-ba54-d1e709610b2d",
+            "version": "KqlParameterItem/1.0",
+            "name": "_WorkbookContext",
+            "label": "Workbook Context",
+            "type": 1,
+            "description": "Whether workbook is opened standalone or as part of the Hub.",
+            "isRequired": true,
+            "query": "print(\"Focused\")",
```

Comment:
this can probably just be a text parameter with a hard coded value of "Focused"

That way you don't need to make a network call to logs just to print the value "Focused"

I think this is in some other workbooks as well so do the same for others